tests/topotests/multicast-pim-sm-topo2/test_multicast_pim_sm_topo2.py

```diff
@@ -0,0 +1,1953 @@
+    """
+    TC_10: Verify SPT switchover working when RPT and SPT path is
+    different
+    """
+
+    tgen = get_topogen()
+    tc_name = request.node.name
+    write_test_header(tc_name)
+
+    # Creating configuration from JSON
+    kill_iperf(tgen)
+    clear_ip_mroute(tgen)
+    reset_config_on_routers(tgen)
+    clear_ip_pim_interface_traffic(tgen, topo)
+
+    # Don"t run this test if we have any failure.
+    if tgen.routers_have_failure():
+        pytest.skip(tgen.errors)
+
+    step("Configure static RP for (226.1.1.1-5) and " "(232.1.1.1-5) in c2")
+
+    _GROUP_RANGE = GROUP_RANGE_2 + GROUP_RANGE_3
+    _IGMP_JOIN_RANGE = IGMP_JOIN_RANGE_2 + IGMP_JOIN_RANGE_3
+
+    input_dict = {
+        "c2": {
+            "pim": {
+                "rp": [
+                    {
+                        "rp_addr": topo["routers"]["c2"]["links"]["lo"]["ipv4"].split(
+                            "/"
+                        )[0],
+                        "group_addr_range": _GROUP_RANGE,
+                    }
+                ]
+            }
+        }
+    }
+
+    result = create_pim_config(tgen, topo, input_dict)
+    assert result is True, "Testcase {} : Failed Error: {}".format(tc_name, result)
+
+    step(
+        "Enable IGMP on FRR1 interface and send IGMP join "
+        "(226.1.1.1-5) and (232.1.1.1-5)"
+    )
+
+    result = config_to_send_igmp_join_and_traffic(
+        tgen, topo, tc_name, "i1", "i1-l1-eth0", _GROUP_RANGE, join=True
+    )
+    assert result is True, "Testcase {}: Failed Error: {}".format(tc_name, result)
+
+    result = iperfSendIGMPJoin(tgen, "i1", _IGMP_JOIN_RANGE, join_interval=1)
+    assert result is True, "Testcase {}: Failed Error: {}".format(tc_name, result)
+
+    step("Send multicast traffic from FRR3 to '226.1.1.1-5'" ", '232.1.1.1-5' receiver")
+
+    result = config_to_send_igmp_join_and_traffic(
+        tgen, topo, tc_name, "i2", "i2-f1-eth0", _GROUP_RANGE, traffic=True
+    )
+    assert result is True, "Testcase {}: Failed Error: {}".format(tc_name, result)
+
+    step("registerRx and registerStopTx value before traffic sent")
+    state_dict = {"c2": {"c2-f1-eth1": ["registerRx", "registerStopTx"]}}
+    state_before = verify_pim_interface_traffic(tgen, state_dict)
+    assert isinstance(
+        state_before, dict
+    ), "Testcase {} : Failed \n state_before is not dictionary \n "
+    "Error: {}".format(tc_name, result)
+
+    result = iperfSendTraffic(tgen, "i2", _IGMP_JOIN_RANGE, 32, 2500)
+    assert result is True, "Testcase {} : Failed Error: {}".format(tc_name, result)
+
+    step(
+        "Verify in FRR3 sending initial packet to RP using"
+        " 'show ip mroute' and mroute OIL is towards RP."
+    )
+
+    result = verify_ip_mroutes(
+        tgen,
+        "f1",
+        "10.0.5.2",
+        _IGMP_JOIN_RANGE,
+        "f1-i2-eth1",
+        ["f1-c2-eth0", "f1-r2-eth3"],
+    )
+    assert result is True, "Testcase {} : " "Failed Error: {}".format(tc_name, result)
+
+    """
```

Comment:
Please uncomment and add reason why a waittime of two minutes is necessary or remove this section.

Reply:
Thank you. Removed commented code.